Modules/Modules/Defining Modules/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+mod front_of_house {
```

Comment:
I'll think about that. The learner experience isn't very smooth indeed, given that we have no way to say just "Compile".

Reply:
Ok, I've found a better solution: I'll use custom Run Configurations in these tasks. I'll build corresponding crates instead of running the binaries.